plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionFields.java

```diff
@@ -119,4 +137,12 @@ private static String toPartitionField(PartitionSpec spec, PartitionField field)
 
         throw new UnsupportedOperationException("Unsupported partition transform: " + field);
     }
+
+    private static String toIdentifier(String column)
```

Comment:
Add unit tests for this method
eg showing that non-lowercase values get quoted.

Reply:
We should probably also move `toIdentfier` and `fromIdentifer` it to IcebergUtil or a separate class as we want to reuse them in other places as well like `orc_bloom_filters` and `sort_order`. WDYT?

Reply:
@mdesmet thanks for adding this support.
I'm working on the [issue](https://github.com/trinodb/trino/issues/12447) adding `sort_by` for creating table.
Initially, I didn't consider the quotes, but then @findepi rightly pointed me to this PR/issue, so now adding that support similar to what you are doing for the partitions.

Using same methods as  `toIdentfier` and `fromIdentifer` and adding to the `IcebergUtil` only.
Also moving some of the pattern constants to the IcebergUtil only, which will be common for partiiton and sort spec.
Soo, will start the PR.